app.model.js

```diff
@@ -52,4 +52,12 @@ exports.updateArticleVotes = (newVote, article_id)=>{
                     RETURNING *;`, [newVote, article_id]).then(({rows})=>{
                         return rows[0] ?? Promise.reject(404)
                     })
-}
\ No newline at end of file
+}
+
+exports.removeCommentById = (comment_id)=>{
+    return db.query(`DELETE FROM comments WHERE comment_id=$1`,[comment_id]).then(({rowCount})=>{
+        return rowCount===0 ? Promise.reject(404) : Promise.resolve()
```

Comment:
It might be nice to have a message in the reject so you know what error this is when you use this endpoint later